tests/garage/torch/modules/test_gaussian_mlp_module.py

```diff
@@ -0,0 +1,319 @@
+import pickle
+from garage.torch.modules.gaussian_mlp_module import GaussianMLPModule
+import unittest
+from unittest import mock
+import torch
+from torch import nn
+from nose2.tools.params import params
+
+
+class TestGaussianMLPmodule(unittest.TestCase):
+
+    @mock.patch('torch.rand')
+    @params((5, 1, (1,)), (5, 1, (2,)), (5, 2, (3,)), (5, 2, (1, 1)), (5, 3, (2, 2)))
+    def test_std_share_network_output_values(self, input_dim, output_dim, hidden_sizes, mock_normal):
+        mock_normal.return_value = 0.5
+
+        module = GaussianMLPModule(
+            input_dim=input_dim,
+            output_dim=output_dim,
+            hidden_sizes=hidden_sizes,
+            std_share_network=True,
+            hidden_nonlinearity=None,
+            std_parameterization='exp',
+            hidden_w_init=nn.init.ones_,
+            output_w_init=nn.init.ones_
+        )
+
+        action, mean, log_std, std, dist = module(torch.ones(5))
+
+        assert mean.equal(5 * (torch.Tensor(hidden_sizes).prod() + torch.zeros(output_dim)))
```

Comment:
Why are we adding zeros? If they are not useful, we can just remove them.

Reply:
It is more like np.full
torch.Tensor(hidden_sizes).prod() is scalar and torch.zeros(output_dim) is matrix,
so it works like element-wise addition.
But I think this is quite confusing.. so I would change this